msticpy/datamodel/entities/account.py

```diff
@@ -187,4 +205,7 @@ def _create_from_event(self, src_event, role):
         # DisplayName (type System.String)
         "DisplayName": None,
         "ObjectGuid": None,
+        "TimeGenerated": None,
```

Comment:
Are these part of the Account schema?
Seems odd to have timestamps associated with a persistent object like an account

Reply:
https://github.com/microsoft/msticpy/issues/215